tools/ci_build/github/azure-pipelines/nuget/templates/cpu-featurizers.yml

```diff
@@ -1,7 +1,7 @@
 parameters:
   DoEsrp: 'false'
-  PackageNameNative: 'Microsoft.ML.ORTFeaturizers'
-  PackageNameManaged: 'Microsoft.ML.ORTFeaturizers.Managed'
+  PackageNameManaged: 'Microsoft.ML.OnnxRuntime.Managed'
```

Comment:
Technically - we don't want this exposed at all as we don't want this to be overridable ideally.